mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/IndexedRowMatrix.scala

```diff
@@ -113,6 +113,67 @@ class IndexedRowMatrix @Since("1.0.0") (
   }
 
   /**
+    * Converts to BlockMatrix. Creates blocks of `DenseMatrix` with size 1024 x 1024.
+    */
+  def toBlockMatrixDense(): BlockMatrix = {
+    toBlockMatrixDense(1024, 1024)
+  }
+
+  /**
+    * Converts to BlockMatrix. Creates blocks of `DenseMatrix`.
+    * @param rowsPerBlock The number of rows of each block. The blocks at the bottom edge may have
+    *                     a smaller value. Must be an integer value greater than 0.
+    * @param colsPerBlock The number of columns of each block. The blocks at the right edge may have
+    *                     a smaller value. Must be an integer value greater than 0.
+    * @return a [[BlockMatrix]]
+    */
```

Comment:
Add a `@Since` annotation like `toBlockMatrix`. Although I doubt this can make in 2.2.0, you can set it to 2.2.0 temporarily. If there is a suggested version from committers, we can change it later.